### PR TITLE
[14.0][MIG] sale_order_invoicing_finished_task: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT34133

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @carlosdauden @sergio-teruel @Tardo

File: sale_order_invoicing_finished_task/models/project.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    invoiceable = fields.Boolean(
+        string="Invoiceable",
+    )
+    invoicing_finished_task = fields.Boolean(
+        related="sale_line_id.product_id.invoicing_finished_task",
+        readonly=True,
```
Comment (sergio-teruel) [inline sale_order_invoicing_finished_task/models/project.py]: readonly=True for related fields is not needed, is the default value

File: sale_order_invoicing_finished_task/views/project_view.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </record>
+    <record id="edit_project_task_track" model="ir.ui.view">
+        <field name="name">project.task.form.track</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <xpath expr="/form" position="inside">
+                <field name="invoicing_finished_task" invisible="1" />
+            </xpath>
+            <xpath expr="//field[@name='partner_id']/.." position="inside">
```
Comment (sergio-teruel) [inline sale_order_invoicing_finished_task/views/project_view.xml]: Please by change this 

```
            <div name="button_box" position="inside">
                <field name="invoiceable" invisible="1"/>
                <button
                    name="toggle_invoiceable"
                    type="object"
                    attrs="{'invisible': [('invoicing_finished_task','=', False)]}"
                    class="oe_stat_button"
                    icon="fa-file"
                >
                    <span class="text-success" attrs="{'invisible': [('invoiceable', '=', False)]}">Invoiceable</span>
                    <span class="text-danger" attrs="{'invisible': [('invoiceable', '!=', False)]}">Not Invoiceable</span>
                </button>
            </div>

```

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@sergio-teruel changes done

**Comment by pedrobaeza** [comment]
Travis red due to error in upstream Odoo. Fix in https://github.com/odoo/odoo/pull/85269

**Review by carlosdauden — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]
Functional test OK!!

Code review OK!!

**Comment by pedrobaeza** [comment]
CI fixed in #2081

/ocabot merge nobump

**Pull request merged**